<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="firstFormIn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="secondFormIn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="thirdFormIn">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="translationIn">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>50</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>I forma:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>II forma:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>III forma:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Polskie tłumaczenie:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="checkWords">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>180</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Sprawdź</string>
    </property>
   </widget>
   <widget class="QLabel" name="firstFormRight">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="thirdFormRight">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>110</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="secondFormRight">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="translationRight">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>110</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="iconLabel">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>100</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>699</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuUstawienia">
    <property name="title">
     <string>Ustawienia</string>
    </property>
    <addaction name="actionZarz_dzaj_list_s_wek"/>
   </widget>
   <addaction name="menuUstawienia"/>
  </widget>
  <action name="actionZarz_dzaj_list_s_wek">
   <property name="text">
    <string>Zarządzaj listą słówek</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
